packages/mobile/components/TopBar.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="typescript">
+    import { AccountSwitcher, AccountActionsButton, ProfileActionsButton } from './'
+    import features from '../features/features'
+</script>
+
+<div class="flex flex-row justify-between">
```

Comment:
I would update this layout so this doesnt happen, if you update the feature flags the components move around because you are using `flex`
![image](https://user-images.githubusercontent.com/3624944/196942866-674125c7-fbca-4be0-98df-8eba5733f4ec.png)